runtime/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.26)
```

Comment:
Just curious did you need a new feature?

Reply:
Yes the `Development.SABIModule` component was added in 3.26: https://cmake.org/cmake/help/latest/module/FindPython.html